_posts/2015-08-27-quantityqueries.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: "Quantity Queries"
+author: sweta_shrestha
+modified: 2015-08-27
+tags: [ css, quantity queries]
+---
+
+Often within a project,we all have been experiencing the presentational change of our content/layout based on certain requirements. We have been using media queries to change our styles as per the user device. But whenever we want to present our layout differently depending on the number of blocks, we are always thinking of what can be the best option. So, this is where quantity queries appear as a solution to this problem.
```

Comment:
Here at YIPL, we have been using media queries to change our styles as per the user device